packages/headless/src/integration-tests/commerce.test.ts

```diff
@@ -35,19 +34,16 @@ describe.skip('commerce', () => {
         analytics: {
           trackingId: 'barca',
         },
-      },
-      loggerOptions: {level: 'silent'},
-    });
-
-    buildContext(engine, {
-      options: {
-        language: 'en-gb',
-        country: 'gb',
-        currency: 'GBP',
-        view: {
-          url: 'https://sports-dev.barca.group/browse/promotions/surf-with-us-this-year',
+        context: {
+          language: 'en-gb',
```

Comment:
Would it be more representative to adjust to `en` here and capitalize `GB` on the next line?